website/content/en/highlights/2022-02-08-0-20-0-upgrade-guide.md

```diff
@@ -61,3 +65,18 @@ the `events_discarded_total` metric will now look like the following:
 - {"counter":{"value":10.0},"name":"events_discarded_total"... "tags":{"component_id":"foo.first","component_kind":"transform","component_name":"foo.first","component_type":"route"}}
 + {"counter":{"value":10.0},"name":"events_discarded_total"... "tags":{"component_id":"foo","component_kind":"transform","component_name":"foo","component_type":"route","output":"first"}}
 ```
+
+### Deprecations
+
+#### Version 1 of the Vector protocol has been deprecated {#deprecate-v1}
```

Comment:
TIL about named anchors

Reply:
👍 Luc introduced them to me. I think it is a Hugo specific thing.